package.json

```diff
@@ -7,16 +7,16 @@
     "build": "yarn clean && tsc",
     "clean": "rm -rf dist",
     "lint": "tslint --project .",
-    "test": "jest -i",
-    "coverage": "jest -i --coverage",
-    "coverage:local": "export $(cat .env | xargs) && yarn coverage",
+    "test": "IASQL_ENV=test jest -i",
+    "coverage": "IASQL_ENV=test jest -i --coverage",
+    "coverage:local": "IASQL_ENV=test export $(cat .env | xargs) && yarn coverage",
     "start": "if [ \"$IASQL_ENV\" = \"local\" ]; then yarn start:dev; else yarn serve; fi",
     "serve": "node --max_old_space_size=8192 dist/index.js",
     "start:dev": "nodemon -e ts -w ./src -x yarn start:dev:watch",
     "start:dev:watch": "yarn build && yarn serve",
-    "gen-sql": "ts-node ./node_modules/.bin/typeorm migration:generate -n",
-    "migrate": "ts-node ./node_modules/.bin/typeorm migration:run",
-    "gen-module": "./.module_migration.sh",
+    "gen-sql": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:generate -n",
+    "migrate": "IASQL_ENV=local ts-node ./node_modules/.bin/typeorm migration:run",
+    "gen-module": "IASQL_ENV=local ./.module_migration.sh",
     "docker-compose:dev": "docker-compose --env-file .env -f docker-compose.yml -f docker-compose.dev.yml up --build",
```

Comment:
So first, I never used those commands and just run `docker-compose up --build` directly.

The `dev` one in particular doesn't make sense to me because the main `docker-compose.yml` is *already* a dev yaml and has `IASQL_ENV=local` defined within the YAML file.

Second, you *can't* feed docker-compose environment variables that way, they go through the `--env-file` or are hardwired in the docker compose yaml file (as I just mentioned is now the case here). Docker compose just ignores anything directly in the bash environment.

Reply:
oh that works! let's remove or replace for `"docker-compose": "docker-compose up --build"` as a short hand then?

Reply:
I added the `docker-compose:dev` to have hot reload while adding changes, but it is true that it needs the env file or having the variables in the compose file.